packages/atomic/stencil.config.ts

```diff
@@ -107,9 +107,7 @@ export const config: Config = {
       copy: [
         {src: 'pages', keepDirStructure: false},
         {src: 'themes'},
-        isDevWatch
-          ? {src: 'external-builds', dest: 'build/headless'}
-          : {src: ''},
+        {src: 'external-builds', dest: 'build/headless'},
```

Comment:
But does that mean we will now ship this on all our official build 🤔  ?

Reply:
Nope, this is only part of the `www` build.

Reply:
Would just changing insights.html to this achieve the same thing ?
```
await insightInterface.initialize({
          insightId: 'fc2ee6e0-8bda-4883-bfc6-ffc1b0ce34c7',
          accessToken: 'xx9446b04d-45d4-44a6-880c-22c6c04573ff',
          organizationId: 'searchuisamples',
});
```

Reply:
yes, but we also have pages in Atomic that use `external-builds`, it's just that we weren't running e2e tests on them.